src/test/java/NAHW18.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NAHW18 extends BaseTest {
```

Comment:
Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`class HomeWork18`